application-accelerator/creating-accelerators/testing-accelerators.hbs.md

```diff
@@ -0,0 +1,134 @@
+# Testing accelerators
+
+This topic describes how to test an updated accelerator or fragment that is not yet registered in your TAP cluster.
+
+## <a id="testing-accelerators-rapid-iteration"></a>Local Rapid Iteration
+As you are authoring an accelerator, you may wish to try out your accelerator before committing any changes you've made.
+
+With the `tanzu accelerator generate-from-local` command you can run your accelerator, including any changes you have locally,
+specify a set of options and view the generated project.
+
+You can run the accelerator using the components on your TAP cluster, without impacting the state of the TAP cluster.
+
+To do so ensure you have the following prerequisites on your machine:
+
+  - The Tanzu CLI is installed, with the Application Accelerator plugin. For details on installing the Tanzu CLI and plugins, see [Tanzu CLI](../../cli-plugins/tanzu-cli.md)
+  - The server URL is pointing to the TAP cluster you want to test with. For details on setting the server URL, see [Application Accelerator CLI plug-in overview](../../cli-plugins/accelerator/overview.md)
+
+> *Note:* No changes will be made to the TAP cluster that is provided via the server URL. 
+> No new accelerators/fragments will be registered or modified. A TAP cluster is 
+> required in order to ensure consistency between the version that is used for testing 
+> and the version that is used when the accelerator is registered. Furthermore, it allows 
+> using registered fragments and accelerators as dependencies for the local accelerator/fragment.
+
+To use the accelerator that is located on your machine at the path `workspace/java-rest` run the following command:
+
+```console
```

Comment:
Should add an example that tells that options can be passed as a json file at some point IMO

Reply:
In one of the sentences above I point them to the full documentation for `generate-from-local`. I would rather they read about all the options there instead of having it spread across sections.